.gitignore

```diff
@@ -8,3 +8,9 @@ build*
 *.user*
 *.orig
 logs/
+
+
+# CLion Ignore
```

Comment:
I didn't end up making the project work in CLion but figured this couldn't hurt for someone that wants to try in the future